examples/table/sortable-table.html

```diff
@@ -0,0 +1,227 @@
+      </p>
+    </section>
+
+
+    <section>
+      <div class="example-header">
+        <h2 id="ex_label">Example</h2>
+      </div>
+      <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+      <div id="ex1">
+        <table class="sortable">
+          <caption>
+            Students currently enrolled in WAI-ARIA 101
+            <span class="sr-only">, column headers with buttons are sortable.</span>
+          </caption>
+          <thead>
+            <tr>
+              <th>
+                <button>
+                  First Name
+                  <span aria-hidden="true"></span>
+                </button>
+              </th>
+              <th aria-sort="ascending">
+                <button>
+                  Last Name
+                  <span aria-hidden="true"></span>
+                </button>
+              </th>
+              <th>
+                <button>
+                  Company
+                  <span aria-hidden="true"></span>
+                </button>
+              </th>
+              <th class="no-sort">Address</th>
+              <th class="num">
+                <button>
+                  Favorite Number
+                  <span aria-hidden="true"></span>
+                </button>
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td role="cell">Fred</td>
+              <td role="cell">Jackson</td>
+              <td role="cell">Canary, Inc.</td>
+              <td role="cell">123 Broad St.</td>
+              <td role="cell" class="num">56</td>
+            </tr>
+            <tr>
+              <td role="cell">Sara</td>
+              <td role="cell">James</td>
+              <td role="cell">Cardinal, Inc.</td>
+              <td role="cell">457 First St.</td>
+              <td role="cell" class="num">7</td>
+            </tr>
+            <tr>
+              <td role="cell">Ralph</td>
+              <td role="cell">Jefferson</td>
+              <td role="cell">Robin, Inc.</td>
+              <td role="cell">456 Main St.</td>
+              <td role="cell" class="num">513</td>
+            </tr>
+            <tr>
+              <td role="cell">Nancy</td>
+              <td role="cell">Jensen</td>
+              <td role="cell">Eagle, Inc.</td>
+              <td role="cell">2203 Logan Dr.</td>
+              <td role="cell" class="num">3.5</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+    </section>
+
+    <section>
+      <h2>Accessibility Features</h2>
+      <ul>
+        <li>
+          HTML table markup is used to create the rows, caption, and to identify the column headers for the data cells.  The use of HTML supports the ARIA principle of using native element semantics whenever possible.<br/>
```

Comment:
OK with removing this? It is a coding practice, not accessibility feature. As we add more HTML examples, this language will become superfluous.

Reply:
@mcking65 removed the role="cell": from the TD elements of the sortable table example

Reply:
```
          HTML table markup is used to create the rows, caption, and to identify the column headers for the data cells.  The use of HTML supports the ARIA principle of using native element semantics whenever possible.<br/>
```

I don't think we need to say this at all. As we add more and more HTML examples, saying we are using more HTML and less ARIA will be increasingly common and saying this over and over won't add value. Would you be OK with removing thisitem from the list?